### PR TITLE
[EZ-673] Fixed smart image tags generation for multiple images.

### PR DESCRIPTION
**Pull request by Vinay15**

### CODE REVIEW

**Comment by arshadkhan53** [comment]
Hi @Vinay15 Thanks for the pr, While testing this pr i see below two issues

- If we upload multiple media the tags get generated during upload and work well, however after selecting these media if we try to edit any one of the media item and try to add new tag then it fails.
- Another issue is related to theming where select image submit button shift towards far right, and become partially visible.

File: modules/ezcontent_node/modules/ezcontent_smart_article/js/ezcontent_smart_article.js [hunk 1]
```diff
@@ -11,21 +11,27 @@
...
-        $('.field--type-ezcontent-smart-image-tags .ui-autocomplete-input').val(finalTags);
+        tagsField.val(finalTags);
         $(this).addClass('list-disabled');
       });
-      // Handle change event.
-      $('.field--type-ezcontent-smart-image-tags .ui-autocomplete-input', context).on('change mouseleave', function (e) {
+
+      // Handle change event for smart image tags.
+      $('.field--type-ezcontent-smart-image-tags .ui-autocomplete-input', context).on('change mouseleave', function () {
+        console.log('change');
```
Comment (arshadkhan53) [inline modules/ezcontent_node/modules/ezcontent_smart_article/js/ezcontent_smart_article.js]: We can remove this console

File: modules/ezcontent_node/modules/ezcontent_smart_article/js/ezcontent_smart_article.js [hunk 2]
```diff
@@ -11,21 +11,27 @@
    */
   Drupal.behaviors.ezcontent_smart_article = {
     attach: function (context, settings) {
-      $('.field--type-ezcontent-smart-image-tags .image-tag-field-wrapper .tag-wrapper li', context).click(function (e) {
+
+      // For Smart Image tags.
+      $('.image-tag-field-wrapper .tag-wrapper li', context).once().click(function () {
         var tagName = $(this).text();
-        var existingTags = $('.field--type-ezcontent-smart-image-tags .ui-autocomplete-input').val();
+        var imageWrapper = $(this).parent().parent().parent().parent();
```
Comment (arshadkhan53) [inline modules/ezcontent_node/modules/ezcontent_smart_article/js/ezcontent_smart_article.js]: can we use it like this
```suggestion
       var imageWrapper = $(this).parents().eq(3);
```

**Comment by Vinay15** [comment]
> Hi @Vinay15 Thanks for the pr, While testing this pr i see below two issues
> 
> * If we upload multiple media the tags get generated during upload and work well, however after selecting these media if we try to edit any one of the media item and try to add new tag then it fails.
> * Another issue is related to theming where select image submit button shift towards far right, and become partially visible.

@arshadkhan53 Thanks for the review,
both the points were due to theming, which is fixed now. Please check.

**Review by arshadkhan53 — APPROVED** [review]
LGTM

**Pull request merged**